### DISCUSSION TITLE
Problem connecting Zepp CLI zeus bridge with Zepp App

### DISCUSSION THREAD

**Discussion by vsfonseca23** [zepp-health/zeppos-forum#14] [Developer Feedback]
Hi guys,

I'm trying to enable bridge mode on my project in order to test it with android Zepp App by following the tutorial https://docs.zepp.com/docs/1.0/guides/faq/developer-bridge-mode/

I can connect with the Simulator, but not my android Zepp App. 

If I close the Simulator, the command **zeus bridge** warns "No connectable online App or Simulator."

I've already enabled developer mode and bridge on Zepp App, and it says "Connected sucessfully", but it does not connect with.

The command **zeus preview** and QR code reading works fine, but I want to use the bridge for productivity.

The Zepp App version is 7.2.9.

Any ideas?

Thanks

**Comment by mrcoon** [comment]
@vsfonseca23  Please submit feedback via the Zepp APP (note the checkbox on the App Log) and tell us your user ID to help us understand the issue.

**Comment by vsfonseca23** [comment]
@mrcoon I've just submitted via Zepp APP. My user ID is 3087883447. Thanks.

**Comment by tytydraco** [comment]
I am having the same issue. My user ID is: 3088145040.

**Comment by hpcolus** [comment]
I do have the same issue.  I already submitted the log and the picture via ZEPP app.  But it looks nobody has responded here
